btk/obs_conditions.py

```diff
@@ -0,0 +1,117 @@
+from abc import ABC, abstractmethod
+
+import astropy.wcs as WCS
+import descwl
+
+import btk.survey
+
+
+def make_wcs(
+    pixel_scale, shape, center_pix=None, center_sky=None, projection=None, naxis=2
+):
+    """Creates wcs for an image
+
+    Args:
+        pixel_scale (float): pixel size in arcseconds
+        shape (tuple): shape of the image
+        center_pix (tuple): position of the reference pixel used as the center of the
+                            affine transform for the wcs.
+        center_sky (list):
+        naxis (int):
+        projection(str):
+
+    Returns:
+        wcs: WCS
+    """
+    if center_pix is None:
+        center_pix = [(s + 1) / 2 for s in shape]
+    if center_sky is None:
+        center_sky = [0 for _ in range(naxis)]
+    if projection is None:
+        projection = "TAN"
+    w = WCS.WCS(naxis=2)
+    w.wcs.ctype = ["RA---" + projection, "DEC--" + projection]
+    w.wcs.crpix = center_pix
+    w.wcs.cdelt = [pixel_scale for _ in range(naxis)]
+    w.wcs.crval = center_sky
+    w.array_shape = shape
+    return w
+
+
+class WLDObsConditions(ABC):
```

Comment:
Do we need `WLD` in this class name?

Reply:
I wanted to be explicit that this observing conditions are based on the `survey.Survey` object from WeakLensingDeblending, but we can change it if you think that's better. 

We might add other types of observing conditions classes later for the real galsim images for example